fast64_internal/oot/scene/exporter/to_c/scene_table_c.py

```diff
@@ -128,28 +142,35 @@ def sceneTableToC(data, header, sceneNames, scene):
     lastSceneIdx = getInsertionIndex(sceneNames, "SCENE_TESTROOM", None, mode)
 
     # add the actual lines with the same formatting
+    # add the ifdef if this is HackerOoT
     for i in range(len(data)):
         # adds the "// Debug-only scenes"
         # if both lastScene indexes are the same values this means there's no debug scene
         if ((i - 1) == lastNonDebugSceneIdx) and (lastSceneIdx != lastNonDebugSceneIdx):
+            if isHackerOoT:
+                fileData += "\n#ifdef INCLUDE_TEST_SCENES\n"
+
             fileData += "// Debug-only scenes\n"
```

Comment:
Could you externalize this to a "scene_table_debug_scenes_write_header" (or smth) function?

Reply:
I'm not sure to understand 🤔

Reply:
By "externalize" I mean make a new function, call it here, and move stuff from here to that function